tests/vmi_multus_test.go

```diff
@@ -658,8 +658,11 @@ var _ = Describe("[Serial]SRIOV", func() {
 
 		waitVmi := func(vmi *v1.VirtualMachineInstance) {
 			// Need to wait for cloud init to finish and start the agent inside the vmi.
-			tests.WaitAgentConnected(virtClient, vmi)
+			vmi, err := virtClient.VirtualMachineInstance(vmi.Namespace).Get(vmi.Name, &metav1.GetOptions{})
+			Expect(err).ToNot(HaveOccurred())
```

Comment:
As we have only explicit expect in this function (rest are on the invoked ones)
do we want WithOfffest 1 ?
as there are around 7 caller to waitVmi

Reply:
The basic practice is violated here: Do not assert in helpers.
In this case, we are in a helper and this helper calls other helpers which do assert.

As I am not going to fix all these helpers in the PR, I continued the bad pattern.
It is also no more correct to used an offset here because this helper is called also from another helper, not only from tests.
So I am left with giving up and just use the simplest option and hope to refactor this whole deep assertion sometime in the future.